src/test/java/seedu/letsgethired/logic/commands/FindCommandTest.java

```diff
@@ -65,7 +73,8 @@ public void execute_longString_noInternApplicationFound() {
     @Test
     public void execute_multipleKeywords_multipleInternApplicationsFound() {
         String expectedMessage = String.format(MESSAGE_INTERN_APPLICATIONS_LISTED_OVERVIEW, 2);
-        CompanyPartialMatchPredicate predicate = new CompanyPartialMatchPredicate("G");
+        CompanyContainsFieldKeywordsPredicate predicate = new CompanyContainsFieldKeywordsPredicate(
+                Arrays.asList(new Pair<>(PREFIX_COMPANY, "G")));
```

Comment:
The test here seems to be testing for multiple keywords, so perhaps we could change the sample test case to `Apple Jane Street` or something similar instead?